src/components/Jobs/RunningJobs/JobDetails/JobDetails.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {
+  FormattedMessage,
+  FormattedTime,
+  FormattedDate,
+} from 'react-intl';
+import classNames from 'classnames';
+
+import {
+  Progress,
+  jobExecutionPropTypes,
+} from '@folio/stripes-data-transfer-components';
+
+import css from '@folio/stripes-data-transfer-components/lib/Jobs/Job/Job.css';
+
+const checkDateIsToday = dateStr => {
+  return new Date().toDateString() === new Date(dateStr).toDateString();
+};
+
+const formatTime = dateStr => {
+  const isToday = checkDateIsToday(dateStr);
+  const datePart = !isToday && <FormattedDate value={dateStr} />;
+  const timePart = <FormattedTime value={dateStr} />;
+  const todayPart = isToday && <FormattedMessage id="ui-data-export.today" />;
+
+  return <span>{datePart} {timePart} {todayPart}</span>;
+};
+
+const JobDetails = props => {
+  const { job } = props;
+
+  const {
+    jobProfileInfo,
+    fileName,
+    hrId,
+    runBy: {
+      firstName,
+      lastName,
+    },
+    progress: {
+      current,
+      total,
+    },
+    startedDate,
+  } = job;
+
+  return (
+    <>
+      <div className={classNames(css.delimiter, css.jobHeader)}>
+        <span data-test-running-job-profile>{jobProfileInfo.name}</span>
+        <span data-test-running-job-file-name>{fileName}</span>
+      </div>
+
+      <div className={css.delimiter}>
+        <span data-test-running-job-hr-id>{hrId}</span>
+        <FormattedMessage
```

Comment:
Could you please make to be like the following, thus it won't be needed to have the function which is passed to FormattedMessage and make it consistent with the notation [here](https://github.com/folio-org/ui-data-export/pull/11/files#diff-cd1f0e640b3ddcc6a406520eb8bdee28R65).

```suggestion
        <span data-test-running-job-triggered-by>
          <FormattedMessage
            id="ui-data-export.triggeredBy"
            values={{ userName: `${firstName} ${lastName}` }}
          />
        </span>
```